osu.Game.Rulesets.Rush/UI/RushSettingsSubsection.cs

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Shane Woolcock. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Game.Overlays.Settings;
+using osu.Game.Rulesets.Rush.Configuration;
+
+namespace osu.Game.Rulesets.Rush.UI
+{
+    public class RushSettingsSubsection : RulesetSettingsSubsection
+    {
+        protected override string Header => RushRuleset.IsDevelopmentBuild ? "rush (Dev build)" : "rush";
+
+        public RushSettingsSubsection(Ruleset ruleset)
+            : base(ruleset)
+        {
+        }
+
+        [BackgroundDependencyLoader]
+        private void load()
+        {
+            var config = (RushRulesetConfigManager)Config;
+
+            // for an odd reason, Config seems to be passed as null when creating it. doesnt even get called...
```

Comment:
Elaborate?

Reply:
I have no idea, this seems to be something that was a thing in the past, and was copied over without much thought.

It doesn't seem to be needed at all, will remove.